packages/cascara/src/ui/Table/0-Table.mdx

```diff
@@ -278,31 +296,29 @@ With the four props described earlier, we can easily create a table: (open your
       userName: 'Jim Doe',
     },
   ]}
-  dataConfig={{
-    display: [
-      {
-        attribute: 'id',
-        isEditable: false,
-        isLabeled: false,
-        label: 'ID',
-        module: 'number',
-      },
-      {
-        attribute: 'userEmail',
-        isEditable: true,
-        isLabeled: false,
-        label: 'eMail',
-        module: 'text',
-      },
-      {
-        attribute: 'userName',
-        isEditable: true,
-        isLabeled: false,
-        label: 'Name',
-        module: 'text',
-      },
-    ],
-  }}
+  dataDisplay={[
+    {
+      attribute: 'id',
+      isEditable: false,
+      isLabeled: false,
```

Comment:
We not define isLabeled anywhere in our dataDisplay objects.

Could you do a search and find any other places that this is being done to make absolutely sure we do not do this anymore? I keep seeing it so I am worried that this is coming from copy paste, which means developers are going to do it too.

Reply:
The `isEditable` prop was removed from docs. A quick search didnt throw any other unwanted appeareances.

Reply:
I totally misread the prop name 🤣 , `isLabeled` was removed from docs.